python/hsfs/feature_group.py

```diff
@@ -535,6 +537,35 @@ def save_validation_report(
             return self._validation_report_engine.save(report).to_ge_type()
         return self._validation_report_engine.save(report)
 
+    def get_validation_history(
+        self,
+        expectation_id: int,
+        sort_by: Optional[str] = "validation_time:desc",
```

Comment:
Does it make sense if we always return a list sort by time in descending order? User can reverse the order easily in python.